src/test/resources/cardsForStressTests/sendNCards.sh

```
@@ -0,0 +1,36 @@
+
+echo "URL=$url"
+
+
+if [ -z $1 ] || [ -z $2 ]
+then
+    echo "Usage : sendNCards.sh N cardFile , with N being a number, the number of cards send will be N"
+
+else	
+(
+opfab login $url 2002 publisher_test test
+for i in `seq 1 $1`;
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)